triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/ServiceMain.scala

```diff
@@ -197,8 +81,8 @@ object Server {
     }
   }
 
-  case class TriggerActorWithParty(
-      ref: ActorRef[TriggerActor.Message],
+  case class TriggerRunnerWithParty(
```

Comment:
This makes less sense to me than before? it’s not just the runner. It’s a reference to an actor for the runner. Either way, please separate refactorings from the actual change.

Reply:
The reference it contains is now a `TriggerRunner`.